### ISSUE TITLE
Ark+ tutorials feed the model sideways radiographs (transpose never undone)

### ISSUE THREAD

**Issue by atriaybagur** [londonaicentre/FLIP#820]
## Problem

The Ark+ tutorials feed the foundation model **sideways radiographs**. Ark+ is pretrained on upright
chest X-rays, so this is a large distribution shift, and it silently depresses every reported metric.

MONAI's `LoadImaged` returns a DICOM's pixel array **transposed** — indexed `(column, row)` where
`PixelData` is `(row, column)` — so the loaded image comes out rotated 90°. All three Ark+ apps try to
correct that with:

```python
# mt.Rotate90d(keys=["image"], k=-1),
mt.Flipd(keys=["image"], spatial_axis=1),
```

Neither transform undoes a transpose. `Rotate90d(k=-1)` composed with the transpose is algebraically a
plain left–right **mirror** (upright but flipped); the `Flipd` that replaced it mirrors the *sideways*
image, leaving it on its side. The correct inverse of a transpose is a transpose.

Affected:
- `fl-tutorials/nvflare/image_evaluation/arkplus_baseline_classification_evaluation`
- `fl-tutorials/nvflare/image_evaluation/arkplus_multimodel_classification_evaluation`
- `fl-tutorials/nvflare/image_classification/arkplus_fine_tuning`

## Impact — measured

Re-scored the multimodel evaluation holdout sets with the same checkpoints, changing only the
orientation. The pretrained baseline improves substantially; the fine-tuned model is unaffected
(it saturates on this synthetic data).

| Class | UK (n=478) sideways → upright | Thai (n=468) sideways → upright |
| --- | --- | --- |
| Effusion | 0.839 → 0.998 | 0.953 → 1.000 |
| Consolidation | 0.866 → 0.989 | 0.872 → 0.998 |
| Infiltration | 0.846 → 0.858 (n.s.) | 0.875 → 0.878 (n.s.) |
| Lung Nodule or Mass | 0.944 → 0.973 | 0.939 → 0.955 |
| Pneumothorax | 0.642 → 0.972 | 0.832 → 0.997 |
| **Mean** | **0.827 → 0.958** | **0.894 → 0.966** |

Worst case is Pneumothorax on UK: +0.330 AUROC, DeLong p = 3.5e-27. Infiltration is the only class
where the change is not significant.

Because the fine-tuned model is unchanged, the **pretrained-vs-fine-tuned gap** — the headline
comparison in the evaluation — shrinks from 0.172 to 0.041 (UK) and 0.106 to 0.034 (Thai). Fine-tuning
still wins on every class, but most of the reported margin was the baseline being handicapped.

The sideways numbers reproduce the published prod run (model `fcf8cb36…`) exactly, including its
DeLong p-values, so the comparison is like-for-like.

## Acceptance criteria

- [x] All three Ark+ apps undo the loader transpose with `Transposed(indices=(0, 2, 1))`
- [x] The corrected transform is verified against the DICOM as stored, not just visually
- [x] The reason the obvious alternatives (`Rotate90d`, `Flipd`) are wrong is recorded in-code, so it
      is not "fixed" back
- [x] Impact on previously reported Ark+ numbers is written down

## Notes

Anyone holding Ark+ fine-tuned weights trained through the old chain should know they were trained on
sideways images. They score identically upright on this saturated synthetic task, but a retrain is the
clean path.

The same root cause affected the xray classification tutorials, fixed separately in #812 (there the
composition produced a mirror rather than a rotation).
